### PR TITLE
New feature ubtue

### PR DESCRIPTION
**Pull request by helenanebel**

### CODE REVIEW

File: ubtue_Open Journal Systems Standard.js [hunk 1]
```diff
@@ -29,18 +29,19 @@
...
-	else if (url.match(/article/)) return "journalArticle";
+	else if (url.match(/\/article\//)) return "journalArticle";
 	else return false;
 }
 
 function getSearchResults(doc) {
 	var items = {};
 	var found = false;
-	var rows = ZU.xpath(doc, '//*[contains(concat( " ", @class, " " ), concat( " ", "media-heading", " " ))]//a | //*[contains(concat( " ", @class, " " ), concat( " ", "title", " " ))]//a | //a[contains(@href, "/article/view/") and not(contains(@href, "/pdf"))]');
+	var rows = ZU.xpath(doc, '//*[contains(concat( " ", @class, " " ), concat( " ", "media-heading", " " ))]//a | //*[contains(concat( " ", @class, " " ), concat( " ", "title", " " ))]//a | //a[contains(@href, "/article/view/") and not(contains(@href, "/pdf")) and not(contains(., "PDF"))]');
```
[the remark this answers is not in the document]
Comment (helenanebel) [reply ubtue_Open Journal Systems Standard.js:44]: No, it is not a href-match but a match on the text of the element.

[the remark this answers is not in the document]
Comment (jriedl) [reply ubtue_Open Journal Systems Standard.js:44]: Ok. PS: Unfortunately the rest of my comments vanished, so I have to readd them again (see below).

File: ubtue_Open Journal Systems Standard.js [hunk 2]
```diff
@@ -106,7 +107,7 @@ function getOrcids(doc, ISSN) {
   	 // e.g. https://jeac.de/ojs/index.php/jeac/article/view/844
   	 // e.g. https://jebs.eu/ojs/index.php/jebs/article/view/336
   	 // e.g. https://bildungsforschung.org/ojs/index.php/beabs/article/view/783
-  	 if (['2627-6062', "1804-6444", '2748-6419'].includes(ISSN)) {
+  	 if (['2627-6062', "1804-6444", "2748-6419"].includes(ISSN)) {
```
Comment (jriedl) [inline ubtue_Open Journal Systems Standard.js:110]: In most of the palce in this file `'` is used for the ISSNs. Please normalize.

File: ubtue_Wiley Online Library.js [hunk 3]
```diff
@@ -139,7 +139,7 @@ function scrapeBook(doc, url) {
 			'/h6[normalize-space(text())="About The Product"]',
 			'/following-sibling::p'].join(''), null, "\n") || "");
 	newItem.accessDate = 'CURRENT_TIMESTAMP';
-
+	if (item.issue != undefined) item.issue = item.issue.replace(/-/g, "/");
```
Comment (jriedl) [inline ubtue_Wiley Online Library.js:142]: Please extract to a function in all cases (e.g. `normalizeIssue()`)

**Comment by helenanebel** [comment]
Lieber @jriedl ,
ich habe die beiden Fällen nochmal überarbeitet.
Viele Grüße, Helena Nebel

**Comment by jriedl** [comment]
In Kürze auf nu.

**Pull request merged**